### DISCUSSION TITLE
partial pod_template_file

### DISCUSSION THREAD

**Discussion by talna-evogene** [apache/airflow#14922] [Q&A]
Hi
I couldn't find in the docs an example for a pod_template_file.
I'm interested in templating only a subset of the fields in the pod definition while keeping others as the default airflow creates.
Is that possible? 
I would also love to get an example for a valid pod yaml with for this param. 
thanks

**Comment by turbaszek** [comment]
Maybe @dimberman can help here?

**Reply by kameshbrao** [reply]
Iam interested as well @dimberman if you can provide one.
